### ISSUE TITLE
add deconvnet and unet correctness train, val and test metrics as a unit test - facilitates certainty in results

### ISSUE THREAD

**Issue by maxkazmsft** [microsoft/seismic-deeplearning#318]
add performance metrics similar to test/cicd/src/check_performance.py  to quantify whether the models are training correctly on train and test sets on dummy synthetic checkerboard dataset

eventually this should be added as a unit test to increase code coverage

Patch-deconvnet: Need to investigate why the metrics are not close to perfect on checkerboard dataset, like with the UNet model – Pixel Accuracy has to be close to 1 on both training and validation set, Class Accuracy has to be close to 1.0, FW IoU, IoU, etc

![image](https://user-images.githubusercontent.com/19535372/83094553-eca0da00-a06f-11ea-974d-f03b2e1368f2.png)

For UNet model, modifying check_performance.py is relatively straightforward - metrics come out near-perfect on checkerboard dataset. But for patch-devonvnet, debugging is needed.
